src/main/java/com/thealgorithms/stacks/DuplicateStack.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.Scanner;
+
+public class DuplicateStack {
+    static int[] St;
+    static int[] dupSt2;
+    static int max;
+    static int top = -1;
+    static int max_a = 50;
+    static int top2 = -1;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter the maximum in the stack");
+        max = scanner.nextInt();
+
+        St = new int[max];
+        dupSt2 = new int[max_a];
```

Comment:
This duplicate array can have the length of _max_ instead of max_a
`dupSt2 = new int[max];`